src/sql/workbench/services/notebook/browser/models/modelFactory.ts

```diff
@@ -20,6 +20,6 @@ export class ModelFactory implements IModelFactory {
 	}
 
 	public createClientSession(options: IClientSessionOptions): IClientSession {
-		return new ClientSession(options);
+		return this.instantiationService.createInstance(ClientSession, options);
```

Comment:
This is left over from an earlier iteration but no reason not to use it in case we want to inject services into the ClientSession in the future